Library/Homebrew/formula_support.rb

```diff
@@ -28,13 +28,14 @@ def valid?
 
   def to_s
     return @explanation unless @explanation.empty?
+    os_name = OS.mac? ? MacOS.os_name : 'macOS'
```

Comment:
I use `MacOS.os_name` before but CI complains that `MacOS` is uninitialized constant. Test always pass locally. So I thought may be it's the case when `OS.mac` is false?